newdocs/source/conf.py

```diff
@@ -0,0 +1,82 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+sys.path.insert(0, os.path.abspath("../.."))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'hazen'
+copyright = '2022, Haris Shuaib'
+author = 'Haris Shuaib'
+
+# The full version, including alpha/beta/rc tags
+release = '0.6.0'
```

Comment:
We should try to make sure we always pull the version number from a single source, otherwise we can end up with the python package labelled having different labels.

I think the best place to get it from for now is setup.py in the root dir, you could try something like:

`from ...setup import __version__`

not sure i have the right number of dots or if a relative path import will work here, but you get the idea. let me know if that doesn't work and we can try something else!